School.js

```diff
@@ -30,6 +30,8 @@ class School {
    */
   enrollStudent(className, student) {
     // Your code here
+      student = new Student(name, age, city, grade)
```

Comment:
This should be `let newStudent = new Student(student.name, student.age, student.city, student.grade) `. You should declare your variable first, then assign it to an instance of Student class, to create which you pass 4 arguments you get from the `student` argument of the current function.